src/Http/Deserializer/Calendar/CalendarJSONParserInterface.php

```diff
@@ -3,34 +3,24 @@
 namespace CultuurNet\UDB3\Http\Deserializer\Calendar;
 
 use CultuurNet\UDB3\Calendar\OpeningHour;
+use CultuurNet\UDB3\Event\ValueObjects\Status;
+use CultuurNet\UDB3\Timestamp;
 
 interface CalendarJSONParserInterface
```

Comment:
Do we even need this interface? There's only one possible implementation I think?

Reply:
Good remark. That was easy to remove.